argilla-server/pyproject.toml

```diff
@@ -57,6 +57,8 @@ dependencies = [
     "typer >= 0.6.0, < 0.10.0", # spaCy only supports typer<0.10.0
     "packaging>=23.2",
     "psycopg2-binary>=2.9.9",
+    # For Telemetry
+    "huggingface_hub>=0.13,<1",
```

Comment:
Why this specific range of versions?

Reply:
Because telemetry was introduced in 0.13, and I want to avoid going to another major release.